Formula/dnsviz.rb

```diff
@@ -44,8 +44,8 @@ def install
       system libexec/"bin/python", "setup.py", "build_ext", "--openssl=#{Formula["openssl"].opt_prefix}", "install"
     end
     venv.pip_install resources.reject { |r| r.name == "m2crypto" }
-    venv.pip_install_and_link buildpath
-    man1.install Dir["doc/man/*.1"]
+    system libexec/"bin/python", "setup.py", "build"
```

Comment:
Was there a problem with `venv.pip_install_and_link buildpath`? This should still use that if possible.